pinot-hadoop/src/main/java/org/apache/pinot/hadoop/job/SegmentPreprocessingJob.java

```diff
@@ -411,7 +408,7 @@ private void setValidationConfigs(Job job, Path path) throws IOException {
       job.getConfiguration().set(InternalConfigConstants.SEGMENT_TIME_FORMAT, _pinotTableSchema.getTimeFieldSpec().getOutgoingGranularitySpec().getTimeFormat());
       job.getConfiguration().set(InternalConfigConstants.SEGMENT_PUSH_FREQUENCY, validationConfig.getSegmentPushFrequency());
       try (DataFileStream<GenericRecord> dataStreamReader = getAvroReader(path)) {
-        job.getConfiguration().set(InternalConfigConstants.TIME_COLUMN_VALUE, (String) dataStreamReader.next().get(timeColumnName));
+        job.getConfiguration().set(InternalConfigConstants.TIME_COLUMN_VALUE, Integer.toString((int) dataStreamReader.next().get(timeColumnName)));
```

Comment:
Use `dataStreamReader.next().get(timeColumnName).toString()` in case time column is not of type INT